benchbuild/utils/validators.py

```diff
@@ -0,0 +1,24 @@
+"""
+    Collection of reusable validators for usage in attr.s.
+"""
+from __future__ import annotations
+
+import collections
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Callable
+
+
+def attribute_not_empty(ATTR: str) -> Callable[..., None]:
```

Comment:
C0103 (invalid-name): Argument name "ATTR" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern  <!-- Automatically posted by Lintly -->